draft-ietf-scitt-architecture.md

```diff
@@ -371,6 +371,156 @@ Issuers SHOULD use consistent decentralized identifiers for all their Statements
 They MAY update their DID manifest, for instance to refresh their signing keys or algorithms, but they SHOULD NOT remove or change any prior keys unless they intend to revoke all Signed Statements that are registered as Transparent Statements issued with those keys.
 This DID appears in the Issuer protected header of Signed Statements' Envelopes, while the version of the key from the manifest used to sign the Signed Statement is written in the `kid` header.
 
+`kid` MUST either be an absolute URL,
+or `iss` MUST also be present in the protected header,
+and `kid` MUST be a URL relative to `iss`.
```

Comment:
```suggestion
`kid` MUST either be an absolute URL,
or a relative URL. Relative URL MUST be
relative to an `iss` value. When relative URL is used,
`iss` MUST also be present in the protected header.
```

I think the requirement conveyed can be clearer.

Reply:
not sure if it should say absolute DID URL and relative DID URL which are terms used in [DID Core](https://www.w3.org/TR/did-core/#relative-did-urls). - depends on the response to a question above.